scripts/generate_cheatsheet.py

```diff
@@ -224,6 +224,11 @@ def run_semgrep_on_example(
         pattern_text = open(config_arg_str).read()
         config.write(_config_to_string(_single_pattern_to_dict(pattern_text, lang)))
         config.flush()
+
+        with open(".semgrepignore", "w") as ignore_out:
```

Comment:
Maybe a comment explaining why, that is if there's no .semgrepignore then semgrep uses one by default which skip tests/ directories?

Reply:
Or is it because we have a .semgrepignore that skip semgrep-core/tests?

Reply:
Also where this .semgrepignore is created? In which directory? Shouldn't we delete it after?

Reply:
@aryx we can hold off on this and just wait for https://github.com/returntocorp/semgrep/issues/4539. I'll close the PR till then